weld/easy_ll/mod.rs

```diff
@@ -109,6 +109,26 @@ pub fn load_library(libname: &str) -> Result<(), LlvmError> {
     }
 }
 
+fn output_llvm_ir(engine: LLVMExecutionEngineRef, module: LLVMModuleRef) -> String {
+    // Keep enough space for the llvm IR being created.
+    let mut v = [8 as i8; 200000];
```

Comment:
Is there any way to check the size to allocate before instead of hard coding it? This may not be big enough for giant programs.

Reply:
this was some stuff I left here by mistake...we just need an arbitrary pointer here which would be changed by llvm...have fixed this in the latest commit.